src/transformers/pipelines.py

```diff
@@ -1324,7 +1421,7 @@ def pipeline(
 
     # Use default model/config/tokenizer for the task if no model is provided
     if model is None:
-        models, config, tokenizer = tuple(targeted_task["default"].values())
+        models, config, tokenizer = [targeted_task["default"][k] for k in ["model", "config", "tokenizer"]]
```

Comment:
this removes a dependency on dict ordering

Reply:
Py36 dicts are ordered (and we might move to Python3.6 "soon") so maybe just document this gotcha, but do not change this? Or do you need it for your tests to pass?

Reply:
In my code, the order changes in py3.5 if the value for config (one of the keys in the dict) is set to `bart-large-cnn`. Since we still support python 3.5, it seems safer (and more explicit) to avoid reliance on dict ordering to me.

Reply:
Ok then